docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_advanced_arming:
+
+***********************************************
+Advanced Arming
+***********************************************
+Vertiq modules can support an advanced arming feature, allowing the user to control the armed state of the module with throttle commands or manually 
+and to configure specific behaviors to occur at armed state transitions. Modules will not react to throttle messages until they have armed, providing improved safety. 
+The configurable behaviors on armed state transitions allow users to easily integrate advanced behaviors into their setup just by controlling the throttle messages they send, 
+simplifying flight controller integration. 
+
+The :ref:`arming_module_support` section below details which Vertiq modules currently support advanced arming.
+
+.. _arming_module_support:
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Module Support for Advanced Arming
+
+	+-------------+------------------------------------+
+	| Module      | Advanced Arming Support            |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support Advanced Arming features.
+
+.. table:: Servo Module Support for Advanced Arming
+
+	+-------------+------------------------------------+
+	| Module      | Advanced Arming Support            |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+Armed States
+===============
+Modules can be in one of two armed states at any time:
+
+* **Armed**: In this state the module will spin when it receives a :ref:`throttle command <manual_throttle>`, so it is not safe to be near a module with an attached propeller when the module is armed.
+* **Disarmed**: In this state the module will NOT spin when it receives a :ref:`throttle command <manual_throttle>`. If using IQUART to control the modules it is still possible that the module can be commanded to spin by other commands, such as by setting the “ctrl_volts” of the Propeller Motor Controller client. Because of this, if you are using IQUART messages besides the typical throttle commands to control the motor you should approach it with caution even when it is disarmed. 

```

Comment:
In this state,

If using IQUART to control the module, 

Instead of Because of this, -> As such,